cms/static/cms/js/modules/cms.modal.js

```diff
@@ -666,6 +672,32 @@
 
                 // update title
                 this.ui.titlePrefix.text(el.text());
+            },
+
+            /**
+             * _deletePlugin removes a plugin once created when clicking
+             * on delete or the close item. If we don't do this, an empty
+             * placeholder is generated
+             * https://github.com/divio/django-cms/pull/4381 will eventually
+             * provide a better solution
+             *
+             * @param hideAfter Boolean
```

Comment:
if we want more that would be then an api change which would require you to go over every call and change it (which in case of a public api is then a matter of a major version (sic!)), which is more hassle than adding another param to the list and then it becomes a mess like `_deletePlugin(true, null, false, null, true)`. sure though it sounds extremely unlikely for this case, but for it's more a matter of principle. don't think we care that much for creating object vs variable memory/speed tradeoff